lib/include/jxl/decode.h

```diff
@@ -263,15 +253,10 @@ typedef enum {
   /** Informative event by @ref JxlDecoderProcessInput
    * "JxlDecoderProcessInput": DC image, 8x8 sub-sampled frame, decoded. It is
    * not guaranteed that the decoder will always return DC separately, but when
-   * it does it will do so before outputting the full frame. @ref
-   * JxlDecoderSetDCOutBuffer must be used after getting the basic image
-   * information to be able to get the DC pixels, if not this return status only
-   * indicates we're past this point in the codestream. This event occurs max
-   * once per frame and always later than @ref JXL_DEC_FRAME and other header
-   * events and earlier than full resolution pixel data.
-   *
-   * @deprecated The DC feature in this form will be removed. For progressive
-   * rendering, @ref JxlDecoderFlushImage should be used.
+   * it does it will do so before outputting the full frame. This return status
+   * only indicates we're past this point in the codestream. This event occurs
+   * max once per frame and always later than @ref JXL_DEC_FRAME and other
+   * header events and earlier than full resolution pixel data.
```

Comment:
We should say here that we never return this event any more.

Reply:
Maybe better to just remove this from the enum. If an application is still subscribing to this event, it's probably better to give them a compile error so they can update their code to do progressive rendering correctly, than it is to keep things 'working' but never actually produce the event...

Reply:
Yes, I agree, it is better to remove it, since it is marked as deprecated anyway.